ManatiCR/artavia
# AR-76 contact page #1

[AR-76](https://manati.atlassian.net/browse/AR-76)
[Figma designs](https://www.figma.com/file/sRoXqR6hyiNvc8moaFeZA6/Artavia-Consultor%C3%ADa)
-
**Steps to test**
- Run `sudo rm -r web && composer update && ./scripts/artavia_local_install.sh`
- Using the menu go to the contact page
- There should be a contact form and a block showing information (see design)
- The recaptcha should be present in this form (As anonymous)
- Fill the form and the recaptcha and send it
- The module should be in the following url: http://127.0.0.1:32768/ (mailhog)
